hermes-api/src/main/java/pl/allegro/tech/hermes/api/endpoints/ConsumerEndpoint.java

```diff
@@ -0,0 +1,26 @@
+package pl.allegro.tech.hermes.api.endpoints;
```

Comment:
I can move it to consumer module but I'm not totally convinced that hermes-api should contain only API from management module...